sdk/core/core-amqp/src/errors.ts

```diff
@@ -684,7 +684,7 @@ export function translate(err: AmqpError | Error): MessagingError | Error {
   }
 
   if (isBrowserWebsocketError(err)) {
-    // Translate browser communication errors during opening handshake to generic SeviceCommunicationError
```

Comment:
This wasn't automated, was it? It's correct, but did you do this one by hand?

Reply:
I should mention - I'm impressed by it but I was curious how that spell checker was so smart?

Reply:
Oh, all I did was go file by file and see the warnings in the console, then do "Quick Fixe(s)". I'll send a screenshot.

Reply:
![image](https://user-images.githubusercontent.com/417016/82144702-45bd7200-9815-11ea-90c9-a73ea4d78e7c.png)